### PR TITLE
Fixes #1143 sort projects by id

### LINKED ISSUE #1143: Order projects by id on projects list
# Problem

The API returns projects ordered by id but it appears that the ordering is variable depending on behaviors taken in the client. We should explicitly order by id and write a test for this specifically.

### PR DESCRIPTION
**Pull request by jderr-mx**
# What's in this PR?

Sorts projects by id

## References
Fixes #1143 

Progress on: #

### CODE REVIEW

**Comment by jderr-mx** [comment]
@JoshSmith This one is ready for review. The build failed on the step `yarn global add phantomjs-prebuilt` in circle-ci. It builds and tests pass on my end. Not sure if I have permissions to trigger a build without pushing.

File: tests/integration/components/project-list-test.js [hunk 1]
```diff
@@ -21,8 +36,27 @@ test('it renders an item for each project in the list', function(assert) {
...
+
+  mockProjectIds.forEach(function(id) {
+    mockProjects.push({ id, title: `Project ${id}`, slug: `project_${id}` });
+  });
+
+  set(this, 'projects', mockProjects);
+  page.render(hbs`{{project-list projects=projects}}`);
+
+  for (let i = 0; i < 3; i++) {
+    let id = i + 1;
```
Comment (filipecrosk) [inline tests/integration/components/project-list-test.js]: I think you could do a little small improvement here, just remove the id line and change the for to `for (let id = 1; id <= 3; id++) {`. 

Nice work! 👍

Comment (jderr-mx) [inline tests/integration/components/project-list-test.js]: Thanks!

I needed the 0 - 2 of `i` for the index of `page.items` but I need 1 - 3 for `id` to assert project id's if that makes sense.

Comment (filipecrosk) [inline tests/integration/components/project-list-test.js]: you're right, my bad, sorry 🙈

**Pull request merged**